apromore/ApromoreCode
# AP-7164 Retain attachments and shapes in re-imported linked subprocess

- Retain attachments, comments, colors, shape sizes and positions when re-creating a subprocess model.
- Hide comments and attachments of elements in a collapsed subprocess

To test:
- Create a model with a subprocess
- Link the subprocess to a model with comments and attachments
- Export the first model with linked subprocessses included
- Check that the subprocess in the newly created model links to a copy of the linked subprocess. This copy should have the same attachments and comments as the original linked subprocess model.
